README.md

```diff
@@ -45,3 +45,12 @@ To generate a new sets of files to reflect your changes please run the command b
 $ dart run build_runner build --delete-conflicting-outputs
 ```
 
+### Proguard rules
```

Comment:
Awesome, let's make sure we link out the issue # on here too, just so that devs have a sample to look at if they run into the same issue. No one ever check closed issues when they run into errors 🙄

Reply:
Lmk how it looks now 👍